pkg/common/telemetry/m3.go

```diff
@@ -11,6 +11,68 @@ import (
 	"github.com/uber-go/tally/m3"
 )
 
+var (
+	// buckets with decreasing granularity as latency increases
+	// given these values, it is much easier to read in this format
+	// than to use the iterative `MakeBuckets` functionality from tally
+	durationBuckets = tally.DurationBuckets{
```

Comment:
this is an initial suggestion for latency buckets, open to suggestions on different bucket cut offs

Reply:
It looks like a decent guess on the face of it. Varying bucket size can get really complicated.